frontend/src/components/registration-row.tsx

```diff
@@ -33,16 +34,46 @@ interface Props {
 const MotionTr = motion<TableRowProps>(Tr);
 
 const RegistrationRow = ({ registration, questions }: Props) => {
-    const { isOpen, onOpen, onClose } = useDisclosure();
+    const { isOpen: isOpenDelete, onOpen: onOpenDelete, onClose: onCloseDelete } = useDisclosure();
 
     const [deleted, setDeleted] = useState(false);
 
+    const [canPromote, setCanPromote] = useState(false);
+
     const toast = useToast();
 
     const router = useRouter();
 
     const { idToken, signedIn } = useAuth();
 
+    useEffect(() => {
+        void CheckIfCanPromote();
+    }, []);
+
+    const CheckIfCanPromote = async () => {
```

Comment:
```suggestion
    const checkIfCanPromote = async () => {
```

camelCase og ikke PascalCase